migrator/migrations/m_192_to_m_193_policy_updates_for_4_3/migration.go

```diff
@@ -0,0 +1,129 @@
+package m192tom193
+
+import (
+	"context"
+	"embed"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/migrations/m_192_to_m_193_policy_updates_for_4_3/schema"
+	"github.com/stackrox/rox/migrator/migrations/policymigrationhelper"
+	"github.com/stackrox/rox/migrator/types"
+	"gorm.io/gorm"
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: 192,
```

Comment:
For your awareness there are at least 3 other PRs out right now using this migration number.  So we will need to keep our eyes open for when those land and adjust if necessary.